### ISSUE TITLE
Enhancement: Wrapper to remove/move with rclone

### ISSUE THREAD

**Issue by TimoPtr** [madslundt/docker-cloud-media-scripts#34]
I didn't find an easy way to move or remove a file or a folder like you did with the upload. It will be nice

**Comment by madslundt** [comment]
Yes that is correct. If you want to move or remove a file/folder it can right now only be done manually with rclone command. I'm looking into it.
